analyzer/compliance-analyzer/src/main/java/eu/fasten/analyzer/complianceanalyzer/ReplacedGCPAuthenticator.java

```diff
@@ -0,0 +1,60 @@
+package eu.fasten.analyzer.complianceanalyzer;
+
+import com.google.auth.oauth2.AccessToken;
+import com.google.auth.oauth2.GoogleCredentials;
+import io.kubernetes.client.util.authenticators.Authenticator;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.Date;
+import java.util.Map;
+
+public class ReplacedGCPAuthenticator implements Authenticator {
+
+    private static final String ACCESS_TOKEN = "access-token";
+    private static final String EXPIRY = "expiry";
+
+    private final GoogleCredentials credentials;
+
+    public ReplacedGCPAuthenticator(GoogleCredentials credentials) {
+        this.credentials = credentials;
+    }
+
+    public String getName() {
+        return "gcp";
```

Comment:
Such hard-coded constants can now be added to `code/data/Constants` class, especially if constant is used more than once. It has just been added to `develop` branch.

Reply:
I rebased on top of [`develop`](https://github.com/fasten-project/fasten/tree/develop) again, but I don't find this `code/data/Constants` class you're talking about. Should it be a `Constants` class under the `eu.faten.analyzer.complianceanalyzer.data` package with only `static` fields?

Reply:
My bad, I put a typo there. It's `core/data/Constants` and yes, it contains only static final fields.

Reply:
Substituted with:
https://github.com/fasten-project/fasten/blob/2800fb55cb57a744e74d14ccf58912df87254bce/analyzer/compliance-analyzer/src/main/java/eu/fasten/analyzer/complianceanalyzer/ReplacedGCPAuthenticator.java#L23-L26
In 2800fb55.